functions/Get-DbaSqlManagementObject.ps1

```diff
@@ -0,0 +1,88 @@
+﻿function Get-DbaSqlManagementObject
+{
+<#
+	.SYNOPSIS
+	Gets SQL Mangaement Object versions installed on the machine.
+
+	.DESCRIPTION
+	 The Get-DbaSqlManagementObject returns an object with the Version and the Add-Type Load Template for each version on the server.
+
+	.PARAMETER VersionNumber
+	This is the specific version number you are looking for. The function will look 
+	for that version only.
+	
+	.PARAMETER Silent
+	Use this switch to disable any kind of verbose messages
+	
+	.NOTES
+	Original Author: Ben Miller (@DBAduck - http://dbaduck.com)
+
+	Tags: SMO
```

Comment:
Just to match format of most other commands can you change this to be just:

```powershell
Tags: SMO
Original Author....
```

No need for the line spacing and such.